ostree-simplified-installer.sh

```diff
@@ -663,7 +782,11 @@ ansible_become_pass=${EDGE_USER_PASSWORD}
 EOF
 
 # Test IoT/Edge OS
-sudo ANSIBLE_STDOUT_CALLBACK=yaml ansible-playbook -v -i "${TEMPDIR}"/inventory -e os_name=redhat -e ostree_commit="${UPGRADE_HASH}" -e ostree_ref="${REF_PREFIX}:${OSTREE_REF}" check-ostree.yaml || RESULTS=0
+if [[ "${ID}-${VERSION_ID}" == "rhel-8.5" ]]; then
```

Comment:
AGAIN! Did you ever tested this on RHEL 8.5??

Reply:
This code change is picked up from main branch, I didn't change it.